src/xdocs/config_whitespace.xml

```diff
@@ -1059,7 +1059,7 @@ import static java.math.BigInteger.ZERO;
           <tr>
             <td>allowLineBreaks</td>
             <td>
-              Control whether whitespace is flagged at linebreaks.
+              Control whitespace is allowed if the token is at a linebreak.
```

Comment:
Shouldn't this be `Control whether is...`?

Reply:
done.